env.sh

```diff
@@ -0,0 +1,76 @@
+cd $HOME/anthos-workshop
+
```

Comment:
I like that this is all centralized but it might be nice to improve the readability by moving all the exports at the top of each major grouping, then doing the echos ent the end of each major grouping.

Reply:
+1, will update it.